From ef5deb02dff7ef7f772db6353990e5ebd3193d52 Mon Sep 17 00:00:00 2001
From: sub77 <sub77@ymail.com>
Date: Sun, 12 Jun 2016 13:02:09 -0500
Subject: [PATCH 2/2] Create emergency account with primary stack phone.

When creating emergency account associate the primary
stack phone with it as emergency calls will be placed
only on primary stack with no sim cards on the device.

Change-Id: I63a9d54950330991e7a93ec8c3982a70d4cdd741
CRs-Fixed: 965269
---
 telephony/java/com/android/internal/telephony/IExtTelephony.aidl | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/telephony/java/com/android/internal/telephony/IExtTelephony.aidl b/telephony/java/com/android/internal/telephony/IExtTelephony.aidl
index 59fbc56..e9be8e1 100644
--- a/telephony/java/com/android/internal/telephony/IExtTelephony.aidl
+++ b/telephony/java/com/android/internal/telephony/IExtTelephony.aidl
@@ -128,5 +128,12 @@ interface IExtTelephony {
     *     '5' - APPSTATE_READY
     */
     int getUiccApplicationState(int slotId, int appIndex);
+ 
+    /**
+    * Get primary stack phone id.
+    * @param - void
+    * @return phone id
+    */
+    int getPrimaryStackPhoneId();
     
 }
-- 
1.9.1

